### PR TITLE
Docs-4214 Update General Settings Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit creates the new UI ref content in GeneralSettingsScreens.md,. Creates an expandable General Settings in Tutorials, and moves article content into new ManageSysConfigSCALE.md, GetSupportScale.md and adds a new support snippet to use for TrueNAS Social Media and Community content, and rewords GeneralSettings.md to cover the other smaller general settings tutorial content. It also add the new EnclosureScreensSCALE.md in UI Ref

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GeneralSettings.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+To allow configuring a non-standard port to access the GUI over HTTPS, enter a port number in the **Web Interface HTTPS Port** field.
+
+Select the cryptographic protocols for securing client/server connections from the **HTTPS Protocols** dropdown list. Select the [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) versions TrueNAS SCALE can use for connection security.
+
+To redirect HTTP connections to HTTPS, select **Web Interface HTTP -> HTTPS Redirect**. A GUI SSL Certificate is required for HTTPS. 
+Activating this also sets the [HTTP Strict Transport Security (HSTS)](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) maximum age to 31536000 seconds (one year). 
+This means that after a browser connects to the web interface for the first time, the browser continues to use HTTPS and renews this setting every year. 
+A warning displays when setting this function. Setting HTTPS redirects can have unintended consequences if an app does not support secure connections. 
+If this occurs, to reset, clear this option and click Save. Then clear the browser cache before trying to connect to the app again.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GeneralSettings.md:38]:
```suggestion
If this occurs, to reset, clear this option and click **Save**. Then clear the browser cache before trying to connect to the app again.
```

File: content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GeneralSettings.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Select the language from the **Language** dropdown list, and then the keyboard layout in **Console Keyboard Map**.
+
+Enter the time zone in **Timezone** and then select the local date and time formats to use.
+
+Click **Save**.
+
+## Adding NTP Servers 
+
+The **NTP Servers** widget allows user to configure Network Time Protocol (NTP) servers, that sync the local system time with an accurate external reference. By default, new installations use several existing NTP servers. TrueNAS SCALE supports adding custom NTP servers.
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GeneralSettings.md:60]:
```suggestion
The **NTP Servers** widget allows users to configure Network Time Protocol (NTP) servers. These sync the local system time with an accurate external reference. By default, new installations use several existing NTP servers. TrueNAS SCALE supports adding custom NTP servers.
```

File: content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GetSupportSCALE.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+{{< toc >}}
+
+There are several options to get support for your TrueNAS installation.
+TrueNAS SCALE users can engage with the TrueNAS community to answer questions and resolve issues. TrueNAS Enterprise hardware customers can also access the fast and effective support directly provided by iXsystems.
+
+TrueNAS SCALE users are welcome to report bugs and vote for or suggest new TrueNAS features in the project Jira instance.
+Have questions? We recommend searching through the software documentation and community resources for answers.
+
+{{< include file="/_includes/content/_includes/UsingTrueNASCommunityAndSocialMedia.md" type="page" >}}
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GetSupportSCALE.md:19]:
```Is this displaying?

{{< include file="/_includes/content/_includes/UsingTrueNASCommunityAndSocialMedia.md" type="page" >}}
```

File: content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GetSupportSCALE.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Complete all available fields and select **Enable iXsystems Proactive Support** if it is not check-marked, then click **Save**.
+
+![GeneralSettingsSCALEProactiveSupportForm](/images/SCALE/GeneralSettingsSCALEProactiveSupportForm.png "SCALE General Settings Proactive Support Form")
+
+## Contacting iXsystems Support
+
+{{< include file="static/includes/General/iXsystemsSupportContact.html.part" html="true" >}}
+
+{{< taglist tag="scalesupport" limit="10" >}}
```
Comment (clairewrtr) [inline content/SCALE/SCALETutorials/SystemSettings/GeneralSettings/GetSupportSCALE.md:104]:
```
As of right now, no related articles are displaying.

{{< taglist tag="scalesupport" limit="10" >}}
```

File: content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+tags:
+ - scaleenclosure
+ - scalepools
+ - scaledisks
+ - scaledashboard
+---
+
+{{< toc >}}
+The **View Enclosure** screen displays an image of the TrueNAS-provided system hardware with drive images you can select. 
+The screen includes information on system pools, disks and their status, details and the HDD stats change with the drive you select on the system image.
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md:15]:
```
Simplify?

The screen includes information on system pools, disks and their status, details and the HDD stats change with the drive you select on the system image.
```

File: content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+Click on a drive to display details for that selected drive and to access the **Identify Drive** option.
+
+![ViewEnclosureR20SelectDisk](/images/SCALE/22.02/ViewEnclosureR20SelectDisk.png "View Enclosure Disk Details R20") 
+
+**Identify Drive** helps you identify which physical drive bay corresponds to the SCALE identification number for that drive. 
+Select the drive, click **Identify Drive** and go to the location of the system server to locate the drive bay with the LED indication turned on to identify the physical drive that corresponds to the software drive location.
+
+Disk details include the pool, drive model and serial number, status, and other options for the selected drive. 
+
+{{< taglist tag="scaleenclosure" limit="10" >}}
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md:61]:
```
As of right now no related "scaleenclosure" articles displaying.

{{< taglist tag="scaleenclosure" limit="10" >}}
```

File: content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+tags:
+ - scalesettings
+ - scaleconsole
+ - scalentp
+ - scalelocalization
+---
+
+{{< toc >}}
+
+The TrueNAS SCALE **System Settings > General** screen includes widgets for **[Support]()**, **[GUI]()**, **[Localization]()**, and **[NTP]()** functions.
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md:15]:
```
These links just link back to this same page? Should these be anchor tags instead to jump to the related content on this page?

The TrueNAS SCALE **System Settings > General** screen includes widgets for **[Support]()**, **[GUI]()**, **[Localization]()**, and **[NTP]()** functions.   
```

File: content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+![GeneralSettingsSCALEProactiveSupportForm](/images/SCALE/GeneralSettingsSCALEProactiveSupportForm.png "SCALE General Settings Proactive Support Form")
+
+{{< /expand >}}
+
+## GUI 
+The **GUI** widget allows users to configure the TrueNAS SCALE web interface address.
+{{< expand "Click Here for More Information" "v" >}}
+Click **Settings** to open the **GUI Settings** screen.
+
+![SystemGeneralGuiSettings](/images/SCALE/22.02/SystemGeneralGuiSettings.png "System General GUI Settings") change
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md:122]:
```
The word "change" is displaying to the right of the image.

![SystemGeneralGuiSettings](/images/SCALE/22.02/SystemGeneralGuiSettings.png "System General GUI Settings")
```

File: content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+By default, new installations use several existing NTP servers. TrueNAS SCALE supports adding custom NTP servers. Click **Add** to open the **Add NTP Server** screen.
+
+![SystemGeneralAddNTPServer](/images/SCALE/22.02/SystemGeneralAddNTPServer.png "Add NTP Server")
+
+| Setting | Description |
+|---------|-------------|
+| **Address** | Enter the hostname or IP address of the NTP server. |
+| **Burst** | Select to use a non-public NTP server. Recommended when **Max Poll** is greater than **10**. Only use on personal NTP servers or those under direct control. Do not enable when using public NTP servers. |
+| **IBurst** | Select to speed up the initial synchronization (seconds instead of minutes). |
+| **Prefer** | Select when using a highly accurate NTP servers such as those with time monitoring hardware. Only use for these highl accurate NTP servers.  |
```
Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md:165]:
```
| **Prefer** | Select when using a highly accurate NTP servers such as those with time monitoring hardware. Only use for these highly accurate NTP servers.  |
```

Comment (clairewrtr) [inline content/SCALE/SCALEUIReference/SystemSettings/GeneralSettingsScreens.md:165]:
```suggestion
| **Prefer** | Select when using a highly accurate NTP servers such as those with time monitoring hardware. Only use for these highly accurate NTP servers.  |
```

**Review by clairewrtr — CHANGES_REQUESTED** [review]
Looks good, just a few minor details.

File: content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md (hunk 5 above)
Comment (micjohnson777) [reply content/SCALE/SCALEUIReference/SystemSettings/EnclosureScreensSCALE.md:15]: I'm going to reword this section as I'm not a fan of what I wrote.

**Comment by micjohnson777** [comment]
Claire, I fixed the broken snippet shortcode, added the missing intra-document links and fixed a few typos. Please review again when you have a mo.

**Review by clairewrtr — APPROVED** [review]

**Pull request merged**